packages/cli/main.js

```diff
@@ -4,19 +4,40 @@ import getopts from "getopts";
 import hrp from "http-request-plus";
 import { format, parse } from "json-rpc-protocol";
 import { mapValues, omit } from "lodash";
-const WebSocket = require("ws");
+import * as WebSocket from "ws";
 
 async function main() {
-  const opts = getopts(process.argv.slice(2));
-  const method = opts._[0];
-  const params = mapValues(omit(opts, "_"), String);
+  async function call(method, params) {
+    const response = parse(
+      await hrp
+        .post("http://localhost:4000/api/", {
+          body: format.request(0, method, params),
+        })
+        .readAll("utf-8")
+    );
+
+    if (response.type === "response") {
+      console.log(response.result);
+    } else if (response.type === "error") {
+      console.error(response.error);
+    }
+  }
 
-  if (params.watch) {
+  let opts = getopts(process.argv.slice(2), {
+    stopEarly: true,
+  });
+
+  if (opts.watch) {
     const websocket = new WebSocket("ws://localhost:4000");
 
     websocket.onerror = event => {
       console.error(event);
     };
+
+    if (opts.watch !== true) {
```

Comment:
`opts.watch` is an option of type `boolean`. It cannot be a name of a method.

Reply:
Let me describe all the possible cases and their result of opts.watch : 
- For `cli createEntry --name Name1` (or another method with its params), `opts.watch` is undefined. 
- For `cli --watch`, sure : `opts.watch=true`.
- But for `cli --watch createEntry --name Name2` (or other method with its options), `opts.watch=createEntry` and to be clearer, `opts = { _: [], watch: "createEntry", name: "Name2" }`

Reply:
You can configure `getopts` to always parse `watch` as a boolean.